MerlinAU.sh

```diff
@@ -1987,17 +1986,18 @@ Please manually update to version $minimum_supported_version or higher to use th
         fi
     fi
 
-    ##------------------------------------------##
-    ## Modified by ExtremeFiretop [2024-Jan-06] ##
-    ##------------------------------------------##
+    ##----------------------------------------##
+    ## Modified by Martinski W. [2024-Jan-06] ##
+    ##----------------------------------------##
     availableRAM_kb=$(_GetAvailableRAM_KB_)
     Say "Required RAM: ${required_space_kb} KB - Available RAM: ${availableRAM_kb} KB"
     check_memory_and_prompt_reboot "$required_space_kb" "$availableRAM_kb"
 
     routerURLstr="$(_GetRouterURL_)"
     Say "Router Web URL is: ${routerURLstr}"
 
-    if [ "$inMenuMode" = true ]; then
```

Comment:
Here we need to use the "**isInteractive**" variable instead of "**inMenuMode**" because we need the account for the case where the script is called with the "**runnow**" argument during an interactive shell session so it's not always called in "menu mode." IOW, there are 3 variations of the call: 1) via the menu option, 2) via a cron job, and 3) via CLI (i.e. interactive shell).